source/_integrations/google_assistant.markdown

```diff
@@ -278,6 +278,10 @@ Your fulfillment URL may be invalid or unreachable. Recheck the `Fulfillment URL
 When using NGINX, ensure that your `proxy_pass` line *does not* have a trailing `/`, as this will result in errors. Your line should look like:
 
     proxy_pass http://localhost:8123;
+    
+#### Scenes are not available to other household members
+
+As scenes are not linked to an area in Home Assistant they do not inherit a room in Google Home. This stops other users in your Google Home from accessing them. To fix this you can set a room for the scene using `entity_config`
```

Comment:
Is this limited to scenes? This applies to any entity without a room assignment, right?
You can also set rooms in Google Home itself, isn't that sufficient?

Reply:
This is true however scenes are not viewable in the google home app therefore the only way to provide a room is like this. 

This is also mentioned on this [forum thread](https://community.home-assistant.io/t/home-assistant-cloud-google-assistant-scenes-multiusers/77462/2), however it took me a while to fine this. I thought it would make a helpful troubleshooting as I had read the docs a number of times before finding the forum thread.

Reply:
Hmm... sounds weird... there must be a way? Going to play with this myself to see what is going on here